### PR TITLE
automate adding items to project

### PR DESCRIPTION
**Pull request by toshipp**
To migrate the new project, I automate adding new issues or pull requests to the project.

### CODE REVIEW

File: .github/workflows/project_bot.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          GITHUB_TOKEN: ${{ steps.create-iat.outputs.token }}
+        run: |
+          if [ ${{ github.event_name }} = issues ]; then
+            CONTENT_ID=${{ github.event.issue.node_id }}
+          else
+            CONTENT_ID=${{ github.event.pull_request.node_id }}
+          fi
+          item_id="$( gh api graphql -f query='
+            mutation($project:ID!, $contentId:ID!) {
+              addProjectNextItem(input: {projectId: $project, contentId: $contentId}) {
```
Comment (peng225) [inline .github/workflows/project_bot.yaml:45]: `addProjectNextItem` looks deprecated. Is that OK?
https://docs.github.com/en/graphql/reference/mutations#addprojectnextitem

File: .github/workflows/project_bot.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      - name: Get project data
+        env:
+          GITHUB_TOKEN: ${{ steps.create-iat.outputs.token }}
+          ORGANIZATION: topolvm
+          PROJECT_NUMBER: "2"
+        run: |
+          proj_id="$(gh api graphql -f query='
+            query($org: String!, $number: Int!) {
+              organization(login: $org){
+                projectNext(number: $number) {
```
Comment (peng225) [inline .github/workflows/project_bot.yaml:28]: `projectNext` looks deprecated. Is that OK?
https://docs.github.com/en/graphql/reference/objects#projectnext

File: .github/workflows/project_bot.yaml (hunk 1 above)
Comment (toshipp) [reply .github/workflows/project_bot.yaml:45]: Good catch! I will update this.

File: .github/workflows/project-bot.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  pull_request:
+    types:
+      - opened
+jobs:
+  add_item:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Issue an access token
+        uses: cybozu/octoken-action@v1
+        id: create-iat
```
Comment (daichimukai) [inline .github/workflows/project-bot.yaml:15]: memo: "iat" stands for "installation access token"

File: .github/workflows/project-bot.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+      - name: Add an item to project
+        env:
+          GITHUB_TOKEN: ${{ steps.create-iat.outputs.token }}
+        run: |
+          if [ ${{ github.event_name }} = issues ]; then
+            CONTENT_ID=${{ github.event.issue.node_id }}
+          else
+            CONTENT_ID=${{ github.event.pull_request.node_id }}
+          fi
+          item_id="$( gh api graphql -f query='
```
Comment (daichimukai) [inline .github/workflows/project-bot.yaml:43]: `item_id` seems to be unused. How about printing whole response?

Comment (toshipp) [reply .github/workflows/project-bot.yaml:43]: Nice idea.

**Review by peng225 — APPROVED** [review]

**Review by daichimukai — APPROVED** [review]

**Pull request merged**